docs/pullRequestTemplate.md

```diff
@@ -0,0 +1,25 @@
+# Description
```

Comment:
Typically, the filename is `pull_request_template.md`, isn't it? https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository